devDocs/selfSignedBuild.md

```diff
@@ -0,0 +1,70 @@
+# Creating a self-signed build
+
+These instructions are based on Microsoft documentation to [create a self-signed certificate](https://docs.microsoft.com/en-us/windows/msix/package/create-certificate-package-signing).
+
+### Prerequisites
+
+From PowerShell running as administrator, install [PKI](https://github.com/PKISolutions/PSPKI#download-and-install-powershell-pki-module-from-the-powershell-gallery-using-powershell):
+
+```ps1
+Install-Module -Name PSPKI
+```
+
+### Create a self-signed certificate
+
+Using PKI, create a self signed build with a custom name (`FriendlyName`) and publisher (`Subject`).
+Other parameters are determined by [MS docs](https://docs.microsoft.com/en-us/windows/msix/package/create-certificate-package-signing#use-new-selfsignedcertificate-to-create-a-certificate).
+
+From PowerShell:
+```ps1
+New-SelfSignedCertificate -FriendlyName "LocalNVDA" -Type Custom -Subject "CN=Test NVDA Build, O=NVDA Dev, C=US" -KeyUsage DigitalSignature -CertStoreLocation "Cert:\CurrentUser\My" -TextExtension @("2.5.29.37={text}1.3.6.1.5.5.7.3.3", "2.5.29.19={text}")

```

Comment:
With what version of PowerShell this is supposed to work? I'm running Powershell 5.1.14 here, and after executing this command I got:
```
New-SelfSignedCertificate : The term 'New-SelfSignedCertificate' is not recognized as the name of a cmdlet, function, s
cript file, or operable program. Check the spelling of the name, or if a path was included, verify that the path is cor
rect and try again.
At line:1 char:1
+ New-SelfSignedCertificate -FriendlyName "LocalNVDA" -Type Custom -Sub ...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (New-SelfSignedCertificate:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException

```

Reply:
Have you installed the PKI cmdlets in prerequisites?